sandbox/technical_user_guide.md

```diff
@@ -180,7 +181,7 @@ You will receive a `200 OK` response and an access token if your credentials wer
 
 _Token string abbreviated for readability._
 
-You will receive a 401 Unauthorized response if your credentials are invalid or if your token has expired. No additional information is returned with a 401 response. When you receive a 401 response for a token you were just using successfully, you should request a new access token as outlined above.
+You will receive a `401 Unauthorized` response if your credentials are invalid or if your token has expired. No additional information is returned with a `401` response. When you receive a 401 response for a token you were just using successfully, you should request a new access token as outlined above.
```

Comment:
Thanks for formatting the HTTP Status Codes in this section.  It does look like there is one more instance of `401` which should include the proper formatting.

Reply:
Caught, accepted, resolved.